.changes/unreleased/operator-Changed-20250325-224420.yaml

```diff
@@ -0,0 +1,4 @@
+project: operator
+kind: Changed
+body: Update Redpanda chart to 5.9.X.
```

Comment:
This commit is being merged into `main` (v25.1) where, formally, there is no connection between the operator and helm chart, we can remove this log.

Reply:
ok